src/telemetry/exampleContexts.ts

```diff
@@ -134,7 +134,7 @@ export const exampleContexts = (
         events: [event],
       };
 
-      const body = encode("TelemetryEvents", apiData);
```

Comment:
My guess is you need to see a specific "TelemetryEvents" key in this body object vs. just the apiData object?